--- origsrc/wxPython-4.0.7.post2/wx/__version__.py	2019-11-12 03:41:19.000000000 +0000
+++ src/wxPython-4.0.7.post2/wx/__version__.py	2020-07-20 23:08:00.417631900 +0100
@@ -4,6 +4,6 @@ VERSION_STRING    = '4.0.7.post2'
 MAJOR_VERSION     = 4
 MINOR_VERSION     = 0
 RELEASE_NUMBER    = 7
-BUILD_TYPE        = 'release'
+BUILD_TYPE        = 'development'
 
 VERSION = (MAJOR_VERSION, MINOR_VERSION, RELEASE_NUMBER, '.post2')
